weasyl/middleware.py

```diff
@@ -132,30 +163,52 @@ def __call__(self, environ, start_response):
 
 class SentryEnvironmentMiddleware(object):
     def __init__(self, app, dsn, reactor=None):
-        import raven
         self.app = app
-        self.client = raven.Client(dsn)
+        self.client = raven.Client(
+            dsn=dsn,
+            release=raven.fetch_git_sha(os.path.abspath(os.path.join(__file__, '../..'))),
+            processors=[
+                'raven.processors.SanitizePasswordsProcessor',
```

Comment:
By the way, it's not clear to me whether the processors here are intended to make the things on sentry safe enough that sentry can be public-ish or if sentry should be locked down. These processors are definitely not enough to prevent sentry from capturing PII and other sensitive information that can't necessarily _directly_ be used to compromise an account.